warehouse/packaging/models.py

```diff
@@ -57,9 +57,12 @@
 class Role(db.Model):
 
     __tablename__ = "roles"
-    __table_args__ = (Index("roles_user_id_idx", "user_id"),)
+    __table_args__ = (
+        Index("roles_user_id_idx", "user_id"),
+        UniqueConstraint("user_id", "project_id", name="_roles_user_project_uc"),
+    )
 
-    __repr__ = make_repr("role_name", "user_name", "package_name")
+    __repr__ = make_repr("role_name")
```

Comment:
This is weird, it never worked before and I just noticed that it explodes when tests kept breaking when it tried to render the repr.

Reply:
looks like it is a relic of when we had strings for 'user' and 'package' on the model.